### PR TITLE
324 zusammengehoerige serien von bilder erstellen

### LINKED ISSUE #324: Zusammengehörige Serien von Bilder erstellen
- Den Kuratoren die Möglichkeit bieten zusammengehörige Bilder (Speisekarten, Karten (Vorder-, Rückseite)) in einer sortierten zusammengehörgen Serie zu präsentieren
- Ermöglichen diese Serien einmalig mit einer Verschlagwortung zu nutzen.
- Ermöglichen diese Serie immer in einem Block zu finden.
- Serien können auch für Duplikate genutzt werden.

### PR DESCRIPTION
**Pull request by MariusDoe**
Closes #324

### CODE REVIEW

**Comment by baerlach-git** [comment]
sequence of length 1 should automatically be deleted, since they do not serve the purpose that was originally intended for picture sequences and currently also are not marked as length 1 sequences in the picture grid, which can be confusing to the user

**Comment by baerlach-git** [comment]
if you make a sequence of texts and put them into a sequence and then save all of them into the clipboard, you will only see 1 image in the clipboard that is not marked as a sequence, which is especially confusing if you delete this picture, because there will still be a picture left after you pressed the button. This may cause users to think the delete button does not work.

Similarily sequences of texts that are linked to a picture should all be displayed in the picture view so as to not mislead useres into thinking there might be only 1 text linked to the picture

**Review by olschulz — APPROVED** [review]
passt scho

File: projects/bp-gallery/src/components/common/picture-gallery/PictureStats.tsx [hunk 1]
```diff
@@ -34,20 +36,26 @@ const PictureStats = ({ picture, hovered }: PictureStatsProps) => {
             like(isLiked);
           }}
         >
-          {isLiked ? (
-            <ThumbUpAlt
-              fontSize='inherit'
-              color='like'
-              className='cursor-pointer !transition-transform !duration-100 hover:scale-110 scale-105'
-            />
+          {picturesInSequenceCount > 1 ? (
```
Comment (olschulz) [inline projects/bp-gallery/src/components/common/picture-gallery/PictureStats.tsx:39]: hovering over the sequence counter still shows the like button title text

**Review by olschulz — APPROVED** [review]

**Pull request merged**